libs/dh/shared/feature-flags/src/lib/feature-flags.ts

```diff
@@ -40,6 +40,11 @@ const makeFeatureFlags = satisfies<FeatureFlagConfig>();
  *   disabledEnvironments: [DhAppEnvironment.prod],
  * },
  */
-export const dhFeatureFlagsConfig = makeFeatureFlags({});
+export const dhFeatureFlagsConfig = makeFeatureFlags({
+  start_wholesale_process_feature_flag: {
+    created: '15-08-2022',
```

Comment:
nitpick: there's a test in the pipeline that will fail if a feature flag is older than 14 days. There's also a task in our backlog about evaluating whether 14 days is enough and it should be changed to something else. So the "right" thing to do for now if your feature flag "expires" is to set a new `created` value.